app/splash/splash-config.js

```diff
@@ -0,0 +1,11 @@
+function splashState($stateProvider, $urlRouterProvider) {
+    $urlRouterProvider.when('/splash', '/splash');
```

Comment:
I don't think you need this. It is saying, if you're on splash go to splash. Please correct me if i'm wrong.